public/index.html

```diff
@@ -90,27 +114,45 @@
                 </li>
               </ul> <!-- /.dropdown-menu -->
             </li> <!-- /.dropdown -->
-          
-          
-         </ul> <!-- /.nav navbar-nav -->
-            
-            
+
+           </ul> <!-- /.nav navbar-nav -->  
+         <!-- third item in navbar, sign in button -->
+           <ul>
+            <div id="user-container">
+                <div hidden id="user-pic"></div>
+                <div hidden id="user-name"></div>
+                <button class="btn btn-primary navbar-btn " hidden id="sign-out">
+                  Sign-out
+                </button>
+                <button class="btn btn-primary navbar-btn " hidden id="sign-in"> 
```

Comment:
Apparently, `firebase.auth` always chooses one of the two buttons to unhide.

What about the other buttons?